sharding/nodesCoordinator/indexHashedNodesCoordinatorLite_test.go

```diff
@@ -85,63 +108,50 @@ func TestIndexHashedNodesCoordinator_SetNodesConfigFromValidatorsInfoMultipleEpo
 	arguments := createArguments()
 
 	shufflerArgs := &NodesShufflerArgs{
-		NodesShard:           3,
-		NodesMeta:            3,
-		Hysteresis:           hysteresis,
-		Adaptivity:           adaptivity,
-		ShuffleBetweenShards: shuffleBetweenShards,
-		MaxNodesEnableConfig: nil,
+		NodesShard: 3,
+		NodesMeta:  3,
 	}
 	nodeShuffler, _ := NewHashValidatorsShuffler(shufflerArgs)
 	arguments.Shuffler = nodeShuffler
 
 	ihnc, err := NewIndexHashedNodesCoordinator(arguments)
 	require.Nil(t, err)
-	epoch := uint32(1)
 
+	epoch := uint32(1)
 	randomness := []byte("rand seed")
 
-	validatorsInfo := createDummyValidatorsInfo()
+	validatorsInfo1 := createDummyValidatorsInfo()
+	validatorsInfo2 := createDummyValidatorsInfo()
 
-	err = ihnc.SetNodesConfigFromValidatorsInfo(epoch, randomness, validatorsInfo)
+	err = ihnc.SetNodesConfigFromValidatorsInfo(epoch, randomness, validatorsInfo1)
 	require.Nil(t, err)
-	err = ihnc.SetNodesConfigFromValidatorsInfo(epoch+1, randomness, validatorsInfo)
+
+	epochConfig, ok := ihnc.nodesConfig[epoch]
+	assert.True(t, ok)
+	verifyEligibleValidators(t, validatorsInfo1, epochConfig)
+
+	err = ihnc.SetNodesConfigFromValidatorsInfo(epoch+1, randomness, validatorsInfo2)
 	require.Nil(t, err)
 
-	_, ok := ihnc.nodesConfig[epoch+1]
+	epochConfig, ok = ihnc.nodesConfig[epoch+1]
 	assert.True(t, ok)
-	verifyValidatorsPubKeysByIndex(t, validatorsInfo, ihnc.nodesConfig[epoch+1])
+	verifyEligibleValidators(t, validatorsInfo2, epochConfig)
 
-	err = ihnc.SetNodesConfigFromValidatorsInfo(epoch+nodesCoordinatorStoredEpochs, randomness, validatorsInfo)
+	err = ihnc.SetNodesConfigFromValidatorsInfo(epoch+nodesCoordinatorStoredEpochs, randomness, validatorsInfo1)
 	assert.Nil(t, err)
 
 	_, ok = ihnc.nodesConfig[epoch]
 	assert.False(t, ok)
 
-	_, ok = ihnc.nodesConfig[epoch+1]
+	epochConfig, ok = ihnc.nodesConfig[epoch+1]
 	assert.True(t, ok)
-	verifyValidatorsPubKeysByIndex(t, validatorsInfo, ihnc.nodesConfig[epoch+1])
+	verifyEligibleValidators(t, validatorsInfo2, epochConfig)
 
 	validators, err := ihnc.GetAllEligibleValidatorsPublicKeys(epoch)
 	require.Nil(t, validators)
 	require.True(t, errors.Is(err, ErrEpochNodesConfigDoesNotExist))
```

Comment:
In this file, `L169`, you receive the error, but do not check it: `err = ihnc.SetNodesConfigFromValidatorsInfo(epoch, []byte{}, validatorsInfo)`

You can either replace it with `_` or check that is nil

Reply:
:+1: